### PR TITLE
Create page heading

### PR DESCRIPTION
**Pull request by HanaAlhemyari**
> Relates #1

### CODE REVIEW

**Review by Roaa2002-web — CHANGES_REQUESTED** [review]
The word 'WORKSHOW' should be replaced with 'WORKSHOP'

**Comment by Roaa2002-web** [comment]
The name of the css class in the heading needs to be updated so that existing styles in the style.css file can take effect (class="some-heading" should be replaced with class="page-heading").

File: index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Git Workflow Workshop</title>
+</head>
+
+<body>
+
+    <h1 class="some-heading">GIT WORKFLOW WORKSHOW</h1>
```
Comment (Roaa2002-web) [inline index.html]:
```suggestion
    <h1 class="page-heading">GIT WORKFLOW WORKSHOW</h1>
```

Comment (HanaAlhemyari) [inline index.html]:
```suggestion
    <h1 class="some-heading">GIT WORKFLOW WORKSHOP</h1>
```

**Review by Roaa2002-web — CHANGES_REQUESTED** [review]
<h1 class="some-heading">GIT WORKFLOW WORKSHOP</h1>
(class="some-heading" -> class="page-heading")

**Pull request merged**